projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/patchouli_books/book/zh_cn/entries/components/network_components.json

```
@@ -11,19 +11,19 @@
     {
       "type": "crafting",
       "anchor": "diagnostic",
-      "text": "$(item)诊断子程序/$是$(l:machines/security_station)安全站/$中的必需组件。$(br)其负责追踪入侵者的$(l:machines/security_station#hacking)入侵尝试/$。若所有$(item)诊断子程序/$皆被入侵，$(item)安全站/$的防护将失效。",
+      "text": "$(item)诊断子程序/$是$(l:machines/security_station)安全站/$中的必需组件。$(br)负责追踪入侵者的$(l:machines/security_station#hacking)入侵尝试/$。若所有$(item)诊断子程序/$皆被入侵，$(item)安全站/$的防护将失效。",
       "recipe": "pneumaticcraft:diagnostic_subroutine"
     },
     {
       "type": "crafting",
       "anchor": "io_port",
-      "text": "$(item)网络交互接口/$是$(l:machines/security_station)安全站/$中的必需组件。$(br)其是对$(item)安全站/$的入侵尝试的入口点。",
+      "text": "$(item)网络交互接口/$是$(l:machines/security_station)安全站/$中的必需组件。$(br)是对$(item)安全站/$入侵尝试的入口点。",
```

如果省主语的话，不如把“是”也删了
* 下面几行需一并处理
```suggestion
      "text": "$(item)网络交互接口/$是$(l:machines/security_station)安全站/$中的必需组件。$(br)对$(item)安全站/$入侵尝试的入口点。",
```

---

xxx是xxx，是xxx。前面的句号改为逗号，删去br换行。不然要换行的话还是当做一段独立的话，带上主语算了。估计我当时改的时候没考虑这点。